### PR TITLE
Add GetUint and GetUint64 method on gin.context

### PR DESCRIPTION
**Pull request by kaiiak**

### CODE REVIEW

**Comment by kaiiak** [comment]
close #2432

File: context.go [hunk 1]
```diff
@@ -295,6 +303,14 @@ func (c *Context) GetInt64(key string) (i64 int64) {
 	return
 }
 
+// GetInt64 returns the value associated with the key as an unsigned integer.
```
Comment (iflamed) [inline context.go:306]: comment typo error: `GetInt64` to `GetUint64`

Comment (kaiiak) [reply context.go]: Thx, I fixed this.

**Review by appleboy — APPROVED** [review]

**Comment by appleboy** [comment]
I don't know why I can't merge this PR and this branch is out-of-date with the base branch

**Comment by thinkerou** [comment]
@kaiiak please recommit a pr and close this, thanks!

**Pull request closed without merging**